scalding-commons-macros/src/main/scala/com/twitter/scalding/commons/macros/Macros.scala

```diff
@@ -13,21 +13,23 @@
  See the License for the specific language governing permissions and
  limitations under the License.
  */
-package com.twitter.scalding.macros
+package com.twitter.scalding.commons.macros
 
 import scala.language.experimental.{ macros => sMacros }
-
-import com.twitter.scalding.commons.thrift.{ ScroogeOrderedBufferable, TBaseOrderedBufferable }
+import com.twitter.scalding.serialization.OrderedSerialization
+import com.twitter.scalding.commons.thrift.{ ScroogeOrderedSerialization, TBaseOrderedSerialization }
 import com.twitter.scrooge.ThriftStruct
 
-import com.twitter.scalding.commons.macros.impl.TBaseOrderedBufferableImpl
-import com.twitter.scalding.commons.macros.impl.ScroogeOrderedBufferableImpl
+import com.twitter.scalding.commons.macros.impl.TBaseOrderedSerializationImpl
+import com.twitter.scalding.commons.macros.impl.ScroogeOrderedSerializationImpl
+import com.twitter.scalding.commons.macros.impl.ScroogeInternalOrderedSerializationImpl
 
 import org.apache.thrift.TBase
 
 object Macros {
 
-  implicit def toTBaseOrderedBufferable[T <: TBase[_, _]]: TBaseOrderedBufferable[T] = macro TBaseOrderedBufferableImpl[T]
-  implicit def toScroogeOrderedBufferable[T <: ThriftStruct]: ScroogeOrderedBufferable[T] = macro ScroogeOrderedBufferableImpl[T]
+  implicit def toTBaseOrderedSerialization[T <: TBase[_, _]]: TBaseOrderedSerialization[T] = macro TBaseOrderedSerializationImpl[T]
+  implicit def toScroogeOrderedSerialization[T <: ThriftStruct]: ScroogeOrderedSerialization[T] = macro ScroogeOrderedSerializationImpl[T]
+  implicit def toScroogeInternalOrderedSerialization[T]: OrderedSerialization[T] = macro ScroogeInternalOrderedSerializationImpl[T]
```

Comment:
does this one really have anything to do with Scrooge? If so, why is it not in the type parameter?